### PR TITLE
Fix label not in place on BO products page in Filter by categories

### LINKED ISSUE #16106: Display regression on BO product page with the filter by categories
Display regression (develop) on BO product page with the filter by categories

Develop:

capture d'écran_2108

1.7.6.1:

capture d'écran_2109

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix label not in place on BO products page in Filter by categories
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16106
| How to test?  | In Catalog > Products, click on  "Filter by categories"

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16175)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: admin-dev/themes/new-theme/scss/components/_category_tree.scss [hunk 1]
```diff
@@ -91,6 +91,13 @@ ul.category-tree {
     }
   }
 
+  label.form-control-label {
+    color: $gray-medium;
+    width: 100%;
+    padding-left: 15px;
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/_category_tree.scss:97]: Please prefer use rem rather than px

**Review by Progi1984 — COMMENTED** [review]
Small change

File: admin-dev/themes/new-theme/scss/components/_category_tree.scss (hunk 1 above)
Comment (atomiix) [reply admin-dev/themes/new-theme/scss/components/_category_tree.scss:97]: Yes, I thought about doing that but as the list just below this label is placed in a `.col-sm` div which is itself padded by 15px I thought I should keep it that way.
Wdyt?

Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/components/_category_tree.scss:97]: Ok. Logical.

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you @atomiix

**Pull request merged**